test/sql/agg_bookends.sql.in

```diff
@@ -31,3 +31,5 @@ SET timescaledb.enable_optimizations TO true;
 \o
 
 :DIFF_CMD
+
+
```

Comment:
```suggestion
```
Leftover?

Reply:
Yep, I used them to test that the check works.